kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConf.scala

```diff
@@ -46,6 +46,8 @@ object MetricsConf {
     .stringConf
     .transform(_.toUpperCase())
     .toSequence()
+    .checkValue(_.forall(ReporterType.values.map(_.toString).contains),
```

Comment:
use checkValues？

Reply:
`checkValues` is used one of pre-defined set, not a sequence, like EngineType, Either `SPARK_SQL` or `FLINK_SQL`. ReporterType is conf as sequence, like `CONSOLE,JMX`. So I don't use checkValue method.

By the way, I see a lot of sequence use `checkValue` method, maybe we should add new check method called `checkSeqValues`.

Reply:
I found it hard to implement the method `checkSeqValues`. Leave it.